<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIWidget</class>
 <widget class="QWidget" name="AIWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clan</string>
  </property>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>441</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
